pyScienceMode2/rehastim_interface.py

```diff
@@ -332,36 +323,20 @@ def stop_stimulation(self):
         """
         Update a stimulation.
         Warning: only the channel that has been initiated can be updated.
-
-        Parameters
-        ----------
-        stimulation_duration: float
-            Time of the stimulation after the update.
-        upd_list_channels: list[Channel]
-            List of the channels that will be updated
         """
         self.amplitude = [0] * len(self.list_channels)
```

Comment:
Store amplitude in a temporary variable to reset self.amplitude to initial values: 
tmp_amp = self.amplitude
self.amplitude = 0
self._sendpacket
self._get_last_ack
self.amplitude = tmp_amp

Reply:
Done.